languages/cpp/exercises/concept/numbers/.docs/introduction.md

```diff
@@ -0,0 +1,73 @@
+There are two different types of numbers in C++: integers and floating point types.
```

Comment:
This introduction has got tons of information. The information is great, but we strive to keep the introduction.md as short as can be (see [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd). To keep it as short as possible, the following guidelines can help:

- The information provided should give the student just enough context to figure out the solution themselves.
- Only information that is needed to understand the fundamentals of the concept and solve the exercise should be provided. Extra information should be left for the after.md.

Could you take a second look at the information in this introduction.md and look to shorten it a bit?

Reply:
IMO, for the purpose of self-learning, we should mention that numeric types in C++ are called "arithmetic types"; lets the student search more accurately for extra information

Reply:
> lets the student search more accurately for extra information

This is indeed a very important point! It's even in [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd):

> Proper technical terms should be used so that the student can easily search for more information.